### ISSUE TITLE
Front facing camera not working on Google Nexus S

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#184]
```
What steps will reproduce the problem?
Last version of SDK\NDK, Google Nexus S(samsung nexus s) updated to last 
Android 2.3

Front facing camera is not working(option is selected in imsdroid main config), 
its just not affects it in any way...during the call always used main camera on 
the back of the phone.
If during the call i select "Switch camera" option from menu video stops for 
about 1 sec and then start again....and it again video from camera on the back 
of the phone...not front facing camera.

p.s. is it possible to make imsdroid always send video by default when video 
call starts ?

```

Original issue reported on code.google.com by `yes....@gmail.com` on 21 Feb 2011 at 8:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 28 Feb 2011 at 9:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi - Even i face the same problem with Google Nexus S. When I want to do a 
video call using yahoo messenger or Tango, the back camera is turning on by 
default and I am unable to swith it to the front camera.

Guys, can you please help fixing the problem.
```

Original comment by `pavan....@gmail.com` on 15 Mar 2011 at 5:40

**Comment by GoogleCodeExporter** [comment]
```
Same problem here: IMSDroid does not allow me to change cameras on my 'Nexus 
S'. Camera switching does work with Tango.
```

Original comment by `hansvd...@gmail.com` on 25 Mar 2011 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
dear all, even i am having the same problem, while on yahoo messenger. my front 
cam does not wlrk. please help me..............
```

Original comment by `pvipinn...@gmail.com` on 18 Apr 2011 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed by revision 448 for Android 2.3+
```

Original comment by `boss...@yahoo.fr` on 7 Aug 2011 at 4:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 7 Aug 2011 at 4:18
- Changed state: **Fixed**
